redbot/cogs/alias/alias.py

```diff
@@ -100,6 +101,24 @@ async def call_alias(self, message: discord.Message, prefix: str, alias: AliasEn
         )
         await self.bot.process_commands(new_message)
 
+    async def paginate_alias_list(self, ctx: commands.Context, alias_list: list) -> list:
```

Comment:
Improved typing (you need to add `List` to imports from `typing` at the top too):
```suggestion
    async def paginate_alias_list(
        self, ctx: commands.Context, alias_list: List[AliasEntry]
    ) -> None:
```